examples/displayer.rs

```diff
@@ -51,10 +51,15 @@ async fn main() -> Result<()> {
         .await?;
 
     // example 1 - basic table
-    let result: QueryRowsResult = session
-        .query_unpaged("SELECT a, b, c FROM examples_ks.basic", &[])
-        .await?
-        .into_rows_result()?;
+
+    // fetch the data
+    let paging_state: PagingState = PagingState::start();
+    let (res, _) = session
+        .query_single_page("SELECT a, b, c FROM examples_ks.basic", &[], paging_state)
+        .await?;
+    // all the data is in the first page, so there is no need to fetch more pages
```

Comment:
:face_with_diagonal_mouth: It's not what I meant by migrating from `query_unpaged` to `query_single_page`... The reason was that there was an assumption that the returned data may be arbitrarily large, that's why `query_unpaged` is unsuitable. With such assumption, executing `query_single_page` only once is **incorrect**.